include/effects/strip/faneffects.h

```diff
@@ -1007,7 +1009,8 @@ class FireFanEffect : public LEDStripEffect
         Sparking(jsonObject[PTY_SPARKING]),
         bReversed(jsonObject[PTY_REVERSED]),
         bMirrored(jsonObject[PTY_MIRORRED]),
-        Order((PixelOrder)jsonObject[PTY_ORDER])
+        Order((PixelOrder)jsonObject[PTY_ORDER]),
+        bMulticolor(jsonObject[PTY_MULTICOLOR])

```

Comment:
I think we'll have to serialize this in `SerializeToJSON()` as well, if we plan to deserialize it here.